logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/keyingmaterials/UpdateKeyingMaterialsUseCaseTests.kt

```diff
@@ -0,0 +1,101 @@
+package com.wire.kalium.logic.feature.conversation.keyingmaterials
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.StorageFailure
+import com.wire.kalium.logic.data.conversation.MLSConversationRepository
+
+import com.wire.kalium.logic.functional.Either
+import io.mockative.Mock
+import io.mockative.Times
+import io.mockative.any
+import io.mockative.anything
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import io.mockative.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertIs
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class UpdateKeyingMaterialsUseCaseTests {
+
+    @Test
+    fun givenOutdatedListGroups_ThenRequestToUpdateThemPerformed() = runTest {
```

Comment:
I think we are missing a negative test for when we have groups where the key material doesn't need to be updated.

Reply:
Covered related cases in DAO's tests. Added them.